float64 frequency
---
float64 limitSeconds